### ISSUE TITLE
Regression code missing

### ISSUE THREAD

**Issue by SeanTBlack** [gmum/geo-gcn#8]
Hi, I am really interested in your method. I am attempting to apply it to some chemical data of my own.

In your paper, you mentioned that you used this method on regression problems for the ESOL and FreeSolv datasets. Do you have the code for that posted somewhere? There are multiple places in your repository that assume a classification problem.

Thank you!

**Comment by mokosaur** [comment]
Hi, to adapt the code for the regression problems, I believe you only need to remove `log_softmax` in line 45 of `architectures.py` and set `out_dim` to 1, and replace the loss function in line 71 of `train_models.py` to `F.mse_loss`.

If I find time next week, I'll test this code and update the repo.
